components/event-processor/org.wso2.carbon.event.processor.admin/src/main/java/org/wso2/carbon/event/processor/admin/EventProcessorAdminService.java

```diff
@@ -185,7 +185,7 @@ public int compare(Object o1, Object o2) {
                     }
                 });
 
-                if (!isDistributedProcessingEnabled()){
+                if (!isDistributedProcessingEnabled()) {
                     configurationDtos[0].setDeploymentStatus("not-distributed");
```

Comment:
What would be the reason for updating the zeroth element here? 
May be we can move "not-distributed" to a constant.